app/views/posts/index.html.erb

```diff
@@ -1,5 +1,55 @@
-<html>
-  <body>
-    <h1>Showing All Posts</h1>
-  </body>
-</html>
\ No newline at end of file
+<h1>Show All Posts</h1>
+<div class='row g-0 border rounded overflow-hidden flex-md-row gap-4 mb-4 shadow-sm h-md-250 position-relative'>
+  <div class='col-auto d-none d-lg-block'>
+    <%= image_tag(@user.photo) %>
+  </div>
+  <div class='col p-4 d-flex flex-column position-static'>
+    <h2 class='card-title'><%= @user.name %></h2>
+    <div class='d-flex mt-4 justify-content-end'>
+        <p class= 'text-right align-right justify-content-end'> Number of Posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+</div>
+
+<div  class='container border rounded'>
+  <h2>List of Posts</h2>
+  <% @posts.each do |post| %>
+    <div  class='container border rounded'>
+     <h3><%= link_to  post.title , user_post_path(@user.id, post.id) ,class: 'h3' %></h3>
+      
+      <p><%= post.text %></p>
+      <div class='d-flex justify-content-end'>
+          <h5>Likes <%= post.likes_counter %></h5> &nbsp;
+          <h5>Comments <%= post.comments_counter %><h5>
+        </div>
+      <hr />
+      <div>
+        <% post.recent_comments.each do |comment| %>
+          <div class='d-flex pb-2 justify-content-start'>
+            <strong><%= comment.author.name %>: </strong>
+            <p><%= comment.text %></p>
+          </div>
+        <% end %>
+      </div>
+    </div>
```

Comment:
- [ ] just as previous reviewer mentioned, it would be nice to give margin between each post as to match the given wireframe :+1: 
your design :point_down: 
![image](https://github.com/Cosmosha/blog-app/assets/66097371/e9f1afbe-6c79-4699-9b1f-a3a37c993565)
given wireframe
![image](https://github.com/Cosmosha/blog-app/assets/66097371/f56d4b64-c209-4a9f-9f60-7c12e8634a02)